rosie/index.js

```diff
@@ -32,7 +32,25 @@ const extendFactory = (factory, Model) => {
 		return fac;
 	};
 
-	Object.assign(factory, { create, createMany, clearAll, requestPayload });
+	const seed = async (seedName, params) => {
+		const { seeds } = factory.statics;
+		if (seedName === 'all') {
+			return Promise.all(Object.values(seeds).map(seedDefinition => create({ seedDefinition, ...params })));
+		}
+		const seedDefinition = seeds[seedName];
+		return create({ seedDefinition, ...params });
```

Comment:
I think this will behave funny if seedDefinition is undefined

Reply:
Doesn't seedDefinition neeed to default to {} and create needs to be { ...seedDefinition, ...params }

Reply:
yeah also now I'm seeing I messed up and didn't destructure it

Reply:
jinx